R/combine_cleaning_logs.R

```diff
@@ -44,7 +44,8 @@
 
    print(names(list_of_log) |> glue::glue_collapse(" ,")  %>% glue::glue("List of element to combine- ", .))
 
-   output[["cleaning_log"]] <- dplyr::bind_rows(list_of_log_only)
+   output[["cleaning_log"]] <- dplyr::bind_rows(list_of_log_only)|> dplyr::mutate(change_type = NA_character_,
+                                                                                  new_value = NA_character_)
 
    output
```

Comment:
Can we add something to add information to complement the cleaning log (e.g. enumerator id, district, etc)
```
combined_log$cleaning_log <- combined_log$cleaning_log %>% 
  left_join(combined_log$checked_data %>% dplyr::select(enumerator_num, neighbourhood, X_uuid), 
            by = c("uuid" = "X_uuid"))
```

Reply:
I would prefer to make a new function for that, as in the checks_** we have a parameter where you can define what columns you want to include. With this function, I just wanted to combine the logs. But if you still want to include then I will add! No strong preference!

Reply:
We can do another function also.